src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/utils/generalemail/GeneralEmailWrapperUtilsTest.java

```diff
@@ -0,0 +1,73 @@
+     * Confirms that the function clears working data, but leaves the email collection intact.
+     * For GeneralEmailWrapper, working data is used to capture User input from an EXUI event.
+     * For GeneralEmailWrapper, the GeneralEmailCollection is maintained as a record of emails sent.
+     * Tests that functionality consistent for contested and consented cases.
+     *
+     * @param caseDetails the {@link FinremCaseDetails} instance containing data to be cleared.
+     */
+    @ParameterizedTest
+    @MethodSource("provideCaseDetails")
+    void givenContestedCase_whenCreateGeneralEMailSubmitted_thenGeneraLEmailWrapperCleared(FinremCaseDetails caseDetails) {
+
+        List<GeneralEmailCollection> emailCollection = List.of(
+                GeneralEmailCollection.builder()
+                        .value(GeneralEmailHolder.builder()
+                                .generalEmailBody("collection test body")
+                                .generalEmailRecipient("collection recipient")
+                                .generalEmailCreatedBy("collection sender")
+                                .build())
+                        .build()
+        );
+
+        caseDetails.getData().getGeneralEmailWrapper().setGeneralEmailRecipient("working recipient");
+        caseDetails.getData().getGeneralEmailWrapper().setGeneralEmailCreatedBy("working sender");
+        caseDetails.getData().getGeneralEmailWrapper().setGeneralEmailUploadedDocument(new CaseDocument());
+        caseDetails.getData().getGeneralEmailWrapper().setGeneralEmailBody("working test body");
+        caseDetails.getData().getGeneralEmailWrapper().setGeneralEmailCollection(emailCollection);
+
+        GeneralEmailWrapperUtils.setGeneralEmailValuesToNull(caseDetails);
+
+        // Assert correct fields changed to null
+        assertNull(caseDetails.getData().getGeneralEmailWrapper().getGeneralEmailRecipient());
+        assertNull(caseDetails.getData().getGeneralEmailWrapper().getGeneralEmailCreatedBy());
+        assertNull(caseDetails.getData().getGeneralEmailWrapper().getGeneralEmailUploadedDocument());
+        assertNull(caseDetails.getData().getGeneralEmailWrapper().getGeneralEmailBody());
+
+        // Validate the contents of GeneralEmailCollection are untouched
+        assertEquals(1, emailCollection.size());
+        GeneralEmailHolder emailHolder = emailCollection.get(0).getValue();
+        assertEquals("collection test body", emailHolder.getGeneralEmailBody());
+        assertEquals("collection recipient", emailHolder.getGeneralEmailRecipient());
+        assertEquals("collection sender", emailHolder.getGeneralEmailCreatedBy());
+    }
+
+    private static Stream<FinremCaseDetails> provideCaseDetails() {
+        return Stream.of(
+                FinremCaseDetailsBuilderFactory.from(123L, CaseType.CONTESTED).build(),
+                FinremCaseDetailsBuilderFactory.from(456L, CaseType.CONSENTED).build()
+        );
+    }
+}
```

Comment:
Should a new line be added? Same with some of the other files above

Reply:
Where though @dawudgovuk? The new line requirement is only at the end of the file. It would be pointless to put it say after line 72.